cmd/cloud.go

```diff
@@ -247,6 +276,14 @@ This will execute the test on the Load Impact cloud service. Use "k6 login cloud
 		var progressErr error
 		ticker := time.NewTicker(time.Millisecond * 2000)
 		shouldExitLoop := false
+		if showCloudLogs {
```

Comment:
Can you log a debug message here, something like `Connecting to cloud logs server...`? It would make debugging potential issues easier.